core/indexer/data.go

```diff
@@ -44,6 +45,11 @@ type Block struct {
 	PrevHash      string        `json:"prevHash"`
 }
 
+// SignersIndexes is a structure containing block signers
+type SignersIndexes struct {
```

Comment:
both the struct and its member are named the same. Rename the struct to something like _SignersIndexesMarshalFormat_

Reply:
Done.